packages/shared/src/utils/config/platform-config.ts

```diff
@@ -40,18 +40,19 @@ export const useOnLoadTranslation = () => {
             i18n.language = getInitialLanguage();
         }
         const is_english = i18n.language === 'EN';
-
         if (is_english) {
             setLoaded(true);
         } else {
             i18n.store.on('added', () => {
                 setLoaded(true);
             });
         }
-
         return () => i18n.store.off('added');
     }, []);
 
-    initMoment(i18n.language);
+    (async () => {
+        await initMoment(i18n.language);
+    })();
```

Comment:
setLoaded(true) should be called after i18n.store.on('added', () => {}) and initMoment promise is resolved

Reply:
<img width="657" alt="image" src="https://github.com/binary-com/deriv-app/assets/56330681/48ee2809-5a12-42bf-9b3b-93967b115c4f">
I think the `initMoment` should be moved here.

Reply:
There will be circular dependency if i put it there, because `date-time.ts` imports localize from translations. I will find another place